Sources/Fuzzilli/Compiler/Compiler.swift

```diff
@@ -987,11 +987,53 @@ public class JavaScriptCompiler {
 
             if unaryExpression.operator == "typeof" {
                 return emit(TypeOf(), withInputs: [argument]).output
+            } else if unaryExpression.operator == "delete" {
+                let argumentExpression = unaryExpression.argument.expression!
+
+                if case .memberExpression(let memberExpression) = argumentExpression {
```

Comment:
Nit: this might be nicer as a `guard`:
```
            guard case .memberExpression(let memberExpression) = unaryExpression.argument.expression else {
                throw CompilerError.invalidNodeError("unary operator must be applied to a member expression")
            }
```
I think then you also don't need to force-unwrap the expression

Reply:
Implemented